README.md

```diff
@@ -92,24 +98,6 @@ pip install pycaret[full]
   
 <div align="left">
 
-## ⚡ PyCaret Time Series Module (beta)
```

Comment:
Maybe instead of removing this completely, we should reword this and say that it is integrated with the main pycaret. We have a lot of posts that point to this section of the readme for Time series